src/api/spec/models/groups_user_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe GroupsUser do
+  let(:admin_user) { create(:admin_user) }
+  let(:group) { create(:group) }
+  let(:user) { create(:confirmed_user, login: 'eisendieter') }
+
+  before do
+    login(admin_user)
+  end
+
+  it 'creates an AddedToGroupEvent when a user is added to a group' do
```

Comment:
How do you expect this to break?

Reply:
Sorry, I forgot to remove the specs from the previous PR.